jcoin.datacommons.io/etlMapping.yaml

```diff
@@ -58,3 +58,19 @@ mappings:
           - name: cmc_title
             src: title
             fn: set
+  - name: jcoin_saes
```

Comment:
we're missing this section: https://github.com/uc-cdis/gitops-qa/pull/1649/files#diff-4adde167f8faa3dd3902be286f6a6b05d9842ee198e1af2fef3f32a02b220b0eR41-R42

Reply:
ah good catch. Added this. Also - looks like there are different versions of some of the microservices between QA and PROD. I'm assuming those shouldn't change right?

Reply:
if you're talking about the fence version, it should be fine, the version in prod is slightly more recent (bug fixes) than the version in QA